src/language/lang_es.php

```diff
@@ -0,0 +1,643 @@
+  'Start Over' =>
+    'Iniciar',
+  'Window is too small' =>
+    'La ventana es muy pequeña',
+  'For the best CTF experience, please make window size bigger.' =>
+    'Para una mejor experiencia, porfavor agranda tu ventana.',
+  'Thank you.' =>
+    'Gracias.',
+  'Logout' =>
+    'Cerrar Sesión',
+  'Registration' =>
+    'Registro',
+  'Play CTF' =>
+    'Juega CTF',
+  'Rules' =>
+    'Reglas',
+  //Translations for GameboardController
+  'Admin' =>
+    'Admin',
+  'ADMIN' =>
+    'ADMIN',
+  'Navigation' =>
+    'Navegación',
+  'View Mode' =>
+    'Modo Visualización',
+  'View mode' =>
+    'Modo visualización',
+  'Tutorial' =>
+    'Tutorial',
+  'Scoreboard' =>
+    'Puntuaciones',
+  'You' =>
+    'Tú',
+  'Others' =>
+    'Otros',
+  'All' =>
+    'Todos',
+  'Leaderboard' =>
+    'Tabla de lideres',
+  'Announcements' =>
+    'Anuncios',
+  'Teams' =>
+    'Equipos',
+  'Filter' =>
+    'Filtrar',
+  'Activity' =>
+    'Actividad',
+  'Game Clock' =>
+    'Reloj del Juego',
+  //Translations for AdminController
+  'Auto' =>
+    'Auto',
+  'All Categories' =>
+    'Todas las Categorías',
+  'Open' =>
+    'Abierto',
+  'Tokenized' =>
+    'Tokenized',
+  'Hour' =>
+    'Hora',
+  'Hours' =>
+    'Horas',
+  'Used by' =>
+    'Utilizado por',
+  'Used By' =>
+    'Utilizado Por',
+  'Available' =>
+    'Disponible',
+  'Registration Tokens' =>
+    'Registro de Tokens',
+  'Create More' =>
+    'Crear más',
+  'Export Available' =>
+    'Exportar Disponibles',
+  'Not started yet' =>
+    'No inició aún',
+  'Configuration' =>
+    'Configuración',
+  'Tokens' =>
+    'Tokens',
+  'Game Configuration' =>
+    'Configuración del Juego',
+  'OK' =>
+    'OK',
+  'status_' =>
+    'estado_',
+  'On' =>
+    'On',
+  'Off' =>
+    'Off',
+  'Player Names' =>
+    'Nombre de los jugadores',
+  'Players Per Team' =>
+    'Jugadores Por Equipo',
+  'Registration Type' =>
+    'Tipos de Registro',
+  'Strong Passwords' =>
+    'Passwords fuertes',
+  'Team Selection' =>
+    'Selección de Equipos',
+  'Game' =>
+    'Juego',
+  'Scoring' =>
+    'Puntuación',
+  'Progressive Cycle (s)' =>
+    'Ciclos Progresivos',
+  'Refresh Gameboard' =>
+    'Refrescar Panel de Juego',
+  'Default Bonus' =>
+    'Bonus por defecto',
+  'Bases Cycle (s)' =>
+    'Ciclos Base',
+  'Default Bonus Dec' =>
+    'Default Bonus Dec',
+  'Timer' =>
+    'Temporizador',
+  'Server Time' =>
+    'Tiempo de Servidor',
+  'Game Duration' =>
+    'Duración del Juego',
+  'Begin Time' =>
+    'Tiempo de Inicio',
+  'Expected End Time' =>
+    'Tiempo de Finalización Esperado',
+  'Language' =>
+    'Lenguaje',
+  'DELETE' =>
+    'BORRAR',
+  'Delete' =>
+    'Borrar',
+  'No Announcements' =>
+    'No hay Anuncios',
+  'Game Controls' =>
+    'Controles del Juego',
+  'Write New Announcement here' =>
+    'Escribir Nuevo Anuncio aquí',
+  'Create' =>
+    'Crear',
+  'General' =>
+    'General',
+  'Back Up Database' =>
+    'Sacar Back Up de la Base de Datos',
```

Comment:
It could be also "Hacer Copia de Seguridad de la Base de Datos" but to not make it too long, we can just leave it as "Copia de Seguridad"